www/js/app.js

```diff
@@ -166,7 +165,18 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
     
     // Unique identifier of the article expected to be displayed
     var expectedArticleURLToBeDisplayed = "";
-    
+ 
+    // define variable for dark preference for matchMedia
+    var darkPreference = window.matchMedia('(prefers-color-scheme:dark)');
+    if (!window.matchMedia) document.getElementById('appThemeSelect').options[0].style.display = "none";
```

Comment:
Don't test on `window.matchMedia` because every browser supports it, but not every browser supports `prefers-color-scheme`. You need to test on the latter. See https://caniuse.com/prefers-color-scheme vs https://caniuse.com/matchmedia.

Reply:
> Don't test on `window.matchMedia` because every browser supports it, but not every browser supports `prefers-color-scheme`. You need to test on the latter. See https://caniuse.com/prefers-color-scheme vs https://caniuse.com/matchmedia.

Ohk got it!